_integration-schemas/zendesk/ticket_forms.md

```diff
@@ -0,0 +1,98 @@
+---
+tap: "zendesk"
+version: "1.0"
+
+name: "ticket_forms"
+doc-link: https://developer.zendesk.com/rest_api/docs/core/ticket_forms
+singer-schema: https://github.com/singer-io/tap-zendesk/blob/master/tap_zendesk/schemas/ticket_forms.json
+description: |
+  The `ticket_forms` table contains info about the ticket forms in your Zendesk account.
+
+  **Note**: Retrieving ticket form data requires Zendesk Agent or Admin permissions.
+
+replication-method: "Key-based Incremental"
+
+api-method:
+  name: List ticket forms
+  doc-link: https://developer.zendesk.com/rest_api/docs/core/ticket_forms#list-ticket-forms
+
+attributes:
+  - name: "id"
+    type: "integer"
+    primary-key: true
+    description: "The ticket form ID."
+
+  - name: "updated_at"
+    type: "date-time"
+    replication-key: true
+    description: "The time the ticket form was last updated."
+
+  - name: "created_at"
+    type: "date-time"
+    description: "The time the ticket form was created."
+
+  - name: "name"
+    type: "string"
+    description: "The name of the ticket form."
+
+  - name: "raw_name"
+    type: "string"
+    description: "The dynamic content placeholder, if present. If not, the `name` value."
+
+  - name: "display_name"
+    type: "string"
+    description: "The name of the ticket form that is displayed to the end user."
+
+  - name: "raw_display_name"
+    type: "string"
+    description: "The dynamic content placeholder, if present. If not, the `display_name` value."
+
+  - name: "url"
+    type: "string"
+    description: "The url of the ticket form."
+
+  - name: "position"
+    type: "integer"
+    description: "The relative position of the ticket form."
+
+  - name: "active"
+    type: "boolean"
+    description: "If `true`, the ticket form is set to active."
+
+  - name: "default"
+    type: "boolean"
+    description: "If `true`, the ticket form is the default form for the account."
+
+  - name: "end_user_available"
+    type: "boolean"
+    description: "If `true`, the ticket form is visible to the end user"
+
+  - name: "in_all_brands"
+    type: "boolean"
+    description: "If `true`, the ticket form is available for use in all brands on the account"
+
+  - name: "ticket_field_ids"
+    type: "array"
+    description: "IDs of all ticket fields which are in this ticket form."
+    array-attributes:
+      - name: "value"
+        type: "integer"
+        description: "The ID of the ticket field."
+        foreign-key: true
+        table: "ticket_fields"
+
+  - name: "restricted_brand_ids"
+    type: "array"
+    description: "IDs of all brands that this ticket form is restricted to."
+    array-attributes:
+      - name: "value"
+        type: "integer"
+        description: "The ID of the brand."
+        foreign-key: true
+        table: "brands"
+  
```

Comment:
This table won't show up in Zendesk's docs because it doesn't have a closing `---` - [here's the preview, where `ticket_forms` is missing](https://deploy-preview-114--stitchdocs.netlify.com/docs/integrations/saas/zendesk#schema).

In Jekyll, files have to have starting and closing `---` (see line one in this file). These mark the beginning and end of a file's Frontmatter section (which is what this file is in its entirety) and must be present for Jekyll to read the file.

If you add `---` on line 93, this'll be good to go!

(Also, sorry to kick this back to you for such a minor thing - I'm trying to break the habit of fixing people's PRs, and you're the first 😉)

Reply:
Ah, ok, makes sense!  Just committed the change

And no worries on kicking the PR back, we're trying to get in to the same habit for Singer community based PRs, so I'm glad to see it in practice :)